tests/base/feature_importance_utils_test.py

```diff
@@ -55,7 +55,7 @@ def test_pipeline(iris_split_dataset_and_model_single_feature):
     _, test_ds, clf = iris_split_dataset_and_model_single_feature
     feature_importances, fi_type = calculate_feature_importance(clf, test_ds)
     print(feature_importances)
-    assert_that(feature_importances['sepal length (cm)'], equal_to(1))
+    assert_that(feature_importances['sepal length (cm)'], equal_to(1))  # pylint: disable=e1136
```

Comment:
why we need that: # pylint: disable=e1136?

Reply:
Because pylint sucks and specifically they have a bug regarding this